handlers/authentication.go

```diff
@@ -94,7 +94,7 @@ func getTokenStringFromRequest(r *http.Request) string {
 func performKialiAuthentication(w http.ResponseWriter, r *http.Request) bool {
 	// Check if user is already logged in
 	oldToken := getTokenStringFromRequest(r)
-	user, err := config.ValidateToken(oldToken)
+	user, _ := config.ValidateToken(oldToken)
```

Comment:
Are we sure we don't want to check the err argument here?

Reply:
If flow reaches this code, we are under two possible cases:
* User is logging in. In this case, it's OK to ignore, it will always fail.
* User is requesting to extend session. Well... Execution should be aborted because it is clear that there is an issue with the token. Indeed, request will be aborted with a forbidden http code but the precise reason won't be forwarded to the client. The method is not receiving params about if we are in the "extend session" case. I changed it to output a debug message to the logs to, at least, we know why it failed in case we need this info.